docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -187,10 +185,10 @@ Modify your `package.json` commands:
 ```diff
    "scripts": {
 -    "start": "react-scripts start",
--    "build": "react-scripts build",
--    "test": "react-scripts test",
 +    "start": "react-app-rewired start",
+-    "build": "react-scripts build",
 +    "build": "react-app-rewired build",
+-    "test": "react-scripts test",
 +    "test": "react-app-rewired test",
      "eject": "react-scripts eject"
   }
```

Comment:
I think this was correct before.

✅

```diff
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
     "eject": "react-scripts eject"
  }
```

❌

```diff
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-app-rewired start",
-    "build": "react-scripts build",
+    "build": "react-app-rewired build",
-    "test": "react-scripts test",
+    "test": "react-app-rewired test",
     "eject": "react-scripts eject"
  }
```

Fixed in #38747.

Why? This is to make it easier to 1. follow what changed 2. copy and paste the change. This is how git behaves by default.

Reply:
Weird, I don't remember touching these lines at all in this PR—not sure how they got changed.